File: Homeworks/01. C# Fundamentals/03. Operators-and-Expressions/07. Point in a circle/PointInACircle.cs
using System;

class PointInACircle
{
    static void Main()
    {
        double x, y, r, z;
        x = double.Parse(Console.ReadLine());
        y = double.Parse(Console.ReadLine());
        r = 2;
        z = Math.Sqrt((x * x) + (y * y));
        if (z <= r)
        {
            Console.WriteLine("yes {0:0.00}", z);
        }
        else
        {
            Console.WriteLine("no {0:0.00}", z);
        }
    }
}

File: Homeworks/06. JS Fundamentals/04. Operators and Expressions/06. Point in a Circle/PointInACircle.js
function PointInACircle(args) {
    var x = args[0];
    var y = args[1];
    var radius= 2;
    var distance = Math.sqrt(x * x + y * y);
    if (distance <= radius) {
        console.log('yes', distance.toFixed(2));
    }
    else{
        console.log('no', distance.toFixed(2));
    }
}